rock_paper_scissors.py

```diff
@@ -0,0 +1,61 @@
+#!/bin/python3
+print('''Let\'s play a game.
+''')
+def playgame():
+  player = input('Rock (1), paper (2) or scissors (3)?')
+  player = int(player)
+  if player ==1:
+    print ('You choose rock.')
+  elif player ==2:
+    print ('You choose paper.')
+  elif player ==3:
+    print ('You choose scissors.')
+  else:
+    print ('Please type 1, 2, or 3.')
+    playgame()
+  import time
+  time.sleep(1)
+  print('''Nice.
+  ''')
+  time.sleep(2)
+  print('1...')
+  time.sleep(1)
+  print('2...')
+  time.sleep(1)
+  print('''3!
+  ''')
+  time.sleep(1)
+  from random import randint
+  chosen = randint(1,3)
+  if chosen==1:
```

Comment:
At this point we have "magic numbers" representing rock, paper and scissors. It will be easier to read the code if we first define them as constants - in python the convention is to use an upper case variable name. It's not strictly constant as we *could* change it, but convention states we don't. e.g.

```
ROCK = 1
PAPER = 2
SCISSORS = 3
```
and then only use the `ROCK` etc. variables later in the program.